xenomint/types/block.go

```diff
@@ -54,7 +54,7 @@ func (h *SignedBlockHeader) Verify() error {
 	return h.DefaultHashSignVerifierImpl.Verify(&h.BlockHeader)
 }
 
-// Block defines a block including a signed block header and its query list.
+// BPBlock defines a block including a signed block header and its query list.
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported type Block should be of the form "Block ..." (with optional leading article)